robot/src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -27,28 +32,39 @@ public class Drive extends DreadbotSubsystem {
     private final MotorControllerGroup rightMotors;
 
     protected final SlewRateLimiter slewRate;
+    private final DifferentialDriveOdometry odometry;
 
-    public Drive() {
+    private final AHRS gryo;
+
+    public Drive(AHRS gryo) {
         this.frontLeftMotor = new DreadbotMotor(new CANSparkMax(MotorConstants.FRONT_LEFT_MOTOR_PORT, MotorType.kBrushless), "frontLeft");
         this.frontRightMotor = new DreadbotMotor(new CANSparkMax(MotorConstants.FRONT_RIGHT_MOTOR_PORT, MotorType.kBrushless), "frontRight");
         this.backLeftMotor = new DreadbotMotor(new CANSparkMax(MotorConstants.BACK_LEFT_MOTOR_PORT, MotorType.kBrushless), "backLeft");
         this.backRightMotor = new DreadbotMotor(new CANSparkMax(MotorConstants.BACK_RIGHT_MOTOR_PORT, MotorType.kBrushless), "backRight");
-        
+
         frontLeftMotor.setIdleMode(IdleMode.kBrake);
         frontRightMotor.setIdleMode(IdleMode.kBrake);
         backLeftMotor.setIdleMode(IdleMode.kBrake);
         backRightMotor.setIdleMode(IdleMode.kBrake);
 
-        frontLeftMotor.setInverted(true);
-        backLeftMotor.setInverted(true);
+        frontLeftMotor.setInverted(false);
         frontRightMotor.setInverted(false);
```

Comment:
This doesn't look right. I thought the working code had both Left motors inverted, and the motor groups (down online 59) were both set to `false` 
We need to set the values on the motor objects because down in `resetOdometry()` and `getWheelSpeeds()` we are getting values from the individual motors (not the motor group) and we need to use the inverted value consistently whether accessing the individual motors or the motor groups.